src/WarcraftLegacies.Source/Setup/FactionSetup/FrostwolfSetup.cs

```diff
@@ -52,7 +53,11 @@ public static void Setup(PreplacedUnitSystem preplacedUnitSystem)
       Frostwolf.ModObjectLimit(FourCC("oshm"), Faction.UNLIMITED); //Shaman
       Frostwolf.ModObjectLimit(FourCC("ospw"), Faction.UNLIMITED); //Spirit Walker
       Frostwolf.ModObjectLimit(FourCC("o00A"), 6); //Far Seer
-      Frostwolf.ModObjectLimit(FourCC("obot"), Faction.UNLIMITED); //Transport Ship
+
+      Frostwolf.ModObjectLimit(Constants.UNIT_O06T_TAUREN_GLADIATOR_FROSTWOLF, 6);
```

Comment:
The quest based ones should be in the `OnAdd` method of their respective quests.